asset_browser/_browser.py

```diff
@@ -132,7 +133,11 @@ def __init__(self, parent=None):
         self.tableView.setSelectionBehavior(QtWidgets.QAbstractItemView.SelectRows)
         self.tableView.setSelectionMode(QtWidgets.QAbstractItemView.ExtendedSelection)
         self.tableView.setShowGrid(False)
-
+        # self.tableView.setStyleSheet("""
+        #     QTableView::item:selected {
+        #         background-color: transparent;
+        #     }
+        # """)
```

Comment:
This makes the blue highlight go away, but selected cell still has a gray background. Motivation is to make the custom drawn long lineedit not clipped by one of the columns